src/resources/postcss/my-tickets/_orders.pcss

```
@@ -4,26 +4,45 @@
  * @since 5.2.0
  */
 
+ .tribe-tickets__tickets-page-wrapper {
+	border-top: 1px solid var(--tec-color-border-default);
+	padding-top: var(--tec-spacer-5);
+ }
+
 .tribe-tickets {
+
+	.tec-tickets__my_tickets-tickets-list-title-container {
```

```suggestion
	.tec-tickets__my-tickets-tickets-list-title-container {
```
If we follow BEM conventions, we shouldn't have an underscore there. Maybe there's a name that's more explanatory?

---

I think `my-tickets` works. I know exactly what it's talking about and I can't think of anything more suitable. I'll just remove the underscore.

---

I'm also removing the redundant `tickets` in the class name.